src/main/java/com/google/error_analyzer/backend/LogDaoHelper.java

```diff
@@ -0,0 +1,59 @@
+/**Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+package com.google.error_analyzer.backend;
+
+import java.util.*;
+import com.google.common.collect.ImmutableList;
+import org.elasticsearch.search.fetch.subphase.highlight.HighlightBuilder.Field;
+import org.elasticsearch.search.fetch.subphase.highlight.HighlightField;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.search.SearchHits;
+
+public class LogDaoHelper {
+
+    // return ArrayList of hit ids corresponding to given searchhit list
+    public ImmutableList < String > hitId(SearchHit[] searchHits) {
```

Comment:
Is there a reason we are using array instead of arraylist?

Reply:
input is returned value of a elasticsearch function

Reply:
input is returned by elasticSearch function